runtime/tests/tests.rs

```diff
@@ -183,11 +213,64 @@ fn start_era_works() {
 	});
 }
 
+// Test to show that every extrinsic applied will add transfer fee to
+// CurrentEraTransactionRewards (until it's paid out at the end of an era)
 #[test]
-fn staking_genesis_config_works() {
+fn current_era_transaction_rewards_storage_update_works() {
 	let balance_amount = 10_000 * TransactionBaseFee::get();
-	let staked_amount = balance_amount / 5;
+	let mut total_transfer_fee: Balance = 0;
+
+	let runtime_call_1 = Call::GenericAsset(pallet_generic_asset::Call::transfer(CENTRAPAY_ASSET_ID, bob(), 123));
+	let runtime_call_2 = Call::GenericAsset(pallet_generic_asset::Call::transfer(CENTRAPAY_ASSET_ID, charlie(), 456));
+
+	ExtBuilder::default()
+		.initial_balance(balance_amount)
+		.build()
+		.execute_with(|| {
+			let xt_1 = sign(CheckedExtrinsic {
+				signed: Some((alice(), signed_extra(0, 0, None, None))),
+				function: runtime_call_1.clone(),
+			});
+			let xt_2 = sign(CheckedExtrinsic {
+				signed: Some((bob(), signed_extra(0, 0, None, None))),
+				function: runtime_call_2.clone(),
+			});
+
+			initialize_block();
+			start_era(1);
+			advance_session(); // advance a session to trigger the beginning of era 2
```

Comment:
Isn't this starting era 1, not era 2? (referring to comments)

Reply:
The comment should be changed to era 1

Reply:
The comment is correct: `assert_eq!(Staking::current_era(), 2);` passes after `advance_session()`